tests/unit/validations/messages-test.js

```diff
@@ -0,0 +1,10 @@
+import { module, test } from 'qunit';
+import { messages } from 'ember-i18n-changeset-validations/validations/messages';
```

Comment:
This import is throwing the following error

![screen shot 2018-01-10 at 11 36 13 am](https://user-images.githubusercontent.com/51207/34785092-2483971a-f5fe-11e7-860d-731c5bd21731.png)

Reply:
I've tried a few different ways to import the `{ messages }` from my addon, but none seem to work, it's my hope someone else sees how my import/export configuration is setup incorrectly.